src/Ocelot/DependencyInjection/OcelotBuilder.cs

```diff
@@ -55,6 +55,10 @@ public class OcelotBuilder : IOcelotBuilder
         private readonly IServiceCollection _services;
         private readonly IConfiguration _configurationRoot;
 
+        public IServiceCollection Services => _services;
```

Comment:
Why do we need to do this? People can already edit these outside of Ocelot builder if they want to.

Reply:
the extended components can be configured under Ocelot ,@aqa510415008  have implemented the convergence of Ocelot and Orleans here, and Ocelot is the Api gateway of
Orleans .
Ocelot.OrleansHttpGateway